packages/action-menu/test/action-menu.test.ts

```diff
@@ -242,10 +242,10 @@ describe('Action menu', () => {
             'sp-menu[slot="submenu"]'
         ) as Menu;
         const selectedItem = submenu.querySelector(
-            '#selected-item'
+            '.selected-item'
```

Comment:
This points to two items now...should be be more explicit to support reading this in the future?